Scoop/Controller/ProfileViewController.swift

```diff
@@ -310,7 +324,7 @@ class ProfileViewController: UIViewController, ProfileViewDelegate {
     }
     
     private func getUserPreferences() {
-        NetworkManager.shared.currentUser.prompts.forEach({ prompt in
+        user?.prompts.forEach({ prompt in
```

Comment:
Remove the parentheses for this higher-order function.
`user?.prompts.forEach { prompt in
...
} `